src/com/lftechnology/java_training/jan7/FileInput.java

```diff
@@ -0,0 +1,134 @@
+package com.lftechnology.java_training.jan7;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+/**
+ * Write a program called FileInput to read an int, a double, and a String form a text file called "in.txt", and produce the following
+ * output: (Hint use Scanner to read from file) a. The integer read is 12 b. The floating point number read is 33.44 c. The String read is
+ * "Peter" d. Hi! Peter, the sum of 12 and 33.44 is 45.44
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class FileInput {
+	private static int[] intReadValue = new int[10];
+	private static double[] doubleReadValue = new double[10];
+	private static String[] stringReadValue = new String[10];
+	private static int intValueCounter = 0;
+	private static int doubleValueCounter = 0;
+	private static int stringValueCounter = 0;
+	private static TrainingConstantsUtility constantValues = new TrainingConstantsUtility();
+
+	public static void main(String[] args) {
+		// read values from file
+		readFromFile();
+		// display values written in file to console
+		displayValueFromFile();
+	}
+
+	/**
+	 * read data from file and store it according to its type
+	 */
+	private static void readFromFile() {
+		Scanner inputFile = null;
+
+		try {
+			inputFile = new Scanner(new BufferedReader(new FileReader("in.txt")));
+			// loop until the value finishes
+			while (inputFile.hasNext()) {
+				// check if value is int
+				if (inputFile.hasNextInt()) {
+					intReadValue[intValueCounter++] = Integer.parseInt(inputFile.next());
+					// check if value is double
+				} else if (inputFile.hasNextDouble()) {
+					doubleReadValue[doubleValueCounter++] = Double.parseDouble(inputFile.next());
+				} else {
+					stringReadValue[stringValueCounter++] = inputFile.next();
+
+				}
+			}
+
+		} catch (Exception e) {
+			constantValues.LOGGER.info("file not found" + e);
+
+		} finally {
+			inputFile.close();
+		}
+
+	}
+
+	/**
+	 * display value stored in different variable which are according to data types
+	 */
+	private static void displayValueFromFile() {
+		constantValues.LOGGER
+				.log(Level.INFO,
+						"\na.The integer read is {0} \nb.The doubling point number read is {1} \nc.the string read is {2} \nd.Hi! {3} The sum of {4}  and {5}  is {6} ",
+						new Object[] { displayIntegerValue(), displayDoubleValue(), displayStringValue(), displayStringValue(),
```

Comment:
here again, try not call the functions inside the logger , better to assign these function output to a variable and log that variable inside log